tests/test_updater_ng.py

```diff
@@ -172,6 +173,12 @@ def _modify_repository_root(
             )
         )
 
+    def _assert_files(self, roles: List[str]):
+        """Assert that local metadata files exist for 'roles'"""
+        expected_files = [f"{role}.json" for role in roles]
+        client_files = sorted(os.listdir(self.client_directory))
+        self.assertEqual(client_files, expected_files)
```

Comment:
Would this work even if `expected_files` is not sorted?

Reply:
Quite right. Fortunately every caller has expected roles in sorted order. Throwing a `sorted()` might save a future test editor some woes.